code/datums/wounds/types/punctures.dm

```diff
@@ -8,8 +8,6 @@
 	sewn_clotting_rate = 0.01
 	clotting_threshold = 0.2
 	sewn_clotting_threshold = 0.1
-	woundpain = 0
```

Comment:
I'm not sure we want to remove wound pain listings under each wound type. If you go back into datum/wound they're inherited the exact same; But for readability purposes, having it explicitly read as 0 in each file in subtypes of wound is probably better.